docs/publishing.md

```diff
@@ -0,0 +1,105 @@
+# How to publish Jenkins agent deb PPA
+
+This document will talk about how to bootstrap a debian package repo, what are its components, 
+what they do and how to build, test, sign and publish a debian package. 
+https://github.com/canonical/jenkins-agent-deb will be used as an example.
+
+This document is based on https://wiki.debian.org/Packaging/Intro and goes more into details about
+the packaging process. It also tries to explain some hidden convention in the packaging process and
+put the packaging into the context of the Platform Engineering team.
+
+### Install necessary packages
+
+```
+sudo apt install build-essential devscripts debhelper
+```
+
+### Bootstrap the package repo
+
+1. debian/changelog
+
+```
+dch --create -v 1.0-0 --package jenkins-agent
+```
+
+2. debian/control - Contains information about your package as well as version of the underlying
+system used for packaging. Example of debian/control for jenkins-agent
```

Comment:
For my understanding, debian/control is a file? Is there a command to create this file, or does the user need to create one from scratch? 

Similar to my step 1 comment, I think it would be useful to add a verb to this step and put the context on a separate line. Something like:

> 2. Create debian/control
> This file contains information about your package as well as the version of the underlying system used for packaging. The following is an example of debian/control for the jenkins-agent package: